package.json

```diff
@@ -290,6 +291,7 @@
     "react-intl-po": "^2.2.2",
     "react-markdown": "^3.3.0",
     "react-motion": "^0.5.2",
+    "react-paginate": "^7.0.0",
```

Comment:
can we avoid this dep please ?

Reply:
it's ok if we decide to merge it with this dep, bit definitely worth creating a separate issue for creating our own, 
it could be a `pi-ui` component.